semconv/src/main/java/io/opentelemetry/semconv/trace/attributes/SemanticAttributes.java

```diff
@@ -315,8 +321,8 @@ public final class SemanticAttributes {
    */
   public static final AttributeKey<String> FAAS_TRIGGER = stringKey("faas.trigger");
 
-  /** The execution ID of the current function execution. */
-  public static final AttributeKey<String> FAAS_EXECUTION = stringKey("faas.execution");
```

Comment:
Similar here, can you add a deprecated `FAAS_EXECUTION` pointing to `FAAS_INVOCATION_ID`?